pkgs/development/python-modules/pythran/default.nix

```diff
@@ -17,12 +17,15 @@ let
 in buildPythonPackage rec {
   pname = "pythran";
   version = "0.11.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
```

Comment:
@fabaff Why this constraint? I have built this successfully with Python 3.7, I think